microsoftazurevmmanagement.json

```diff
@@ -9,16 +9,17 @@
     "product_name": "Azure Compute",
     "product_version_regex": ".*",
     "publisher": "Splunk",
-    "license": "Copyright (c) 2019-2021 Splunk Inc.",
+    "license": "Copyright (c) 2019-2022 Splunk Inc.",
     "app_version": "2.1.2",
```

Comment:
Should we update App version?

Reply:
@paulnawat We don't have to for FIPS changes. @shiyingtu If the release has other bug fixes then yes we have to update the app's minor version.

Reply:
There is no bug fixes for this PR. But I noticed in previous PRs, the release updates the version automatically.